versioned_docs/version-1.0/tutorial/05-non-fungible-tokens-2.md

```diff
@@ -371,32 +358,30 @@ Now, back to the tutorial!
 
 ## Restricting Access to the NFT Collection
 
-In the NFT Collection, all the functions and fields are public,
-but we do not want everyone in the network to be able to call our `withdraw` function.
-This is where Cadence's second layer of access control comes in.
+In the NFT Collection, we will publish a capability to allow anyone
+to access important functionality for our `Collection`, like `deposit()` and `getIDs()`.
+
+This is where an important layer of access control comes in.
 Cadence utilizes [capability security](../language/capabilities.md),
 which means that for any given object, a user is allowed to access a field or method of that object if they either:
 
 - Are the owner of the object
 - Have a valid reference to that field or method (note that references can only be created from capabilities, and capabilities can only be created by the owner of the object)
 
 When a user stores their NFT `Collection` in their account storage, it is by default not available for other users to access.
-A user's authorized account object (`AuthAccount`, which gives access to private storage)
+A user's authorized account object (`&Account`, which gives access to private storage)
```

Comment:
> authorized account object

Maybe "entitled account"? Only suggesting because earlier it's mentioned that `&Account` is publicly accessible and grants access to /public/ namespace.

Reply:
That is a good point. I'll reword it to make it more clear